packages/flutter/test/material/text_field_test.dart

```diff
@@ -10057,9 +10202,9 @@ void main() {
     await tester.pumpAndSettle();
 
     expect(RendererBinding.instance.mouseTracker.debugDeviceActiveCursor(1), SystemMouseCursors.text);
-    expect(find.text('Copy'), findsOneWidget);
+    expect(find.text('Paste'), findsOneWidget);
```

Comment:
Why does this change from `Copy` to `Paste`?

Reply:
Copy isn't shown if the selection is collapsed.  Previously, right clicking would always select the underlying word, so copy would always be shown.  Now it will be collapsed on Windows and Linux, and copy won't be there.  But paste should always be there.

Reply:
Oh I get it. Thanks for the clarification!